android/java/res/layout/fragment_nft_grid.xml

```diff
@@ -54,6 +54,17 @@
                 android:textColor="@color/brave_wallet_day_night_text_color"
                 app:autoSizeTextType="uniform" />
 
+            <ImageView
+                android:id="@+id/fragment_nft_grid_btn_add_nft"
+                android:layout_width="24dp"
+                android:layout_height="24dp"
+                android:layout_marginEnd="16dp"
+                app:srcCompat="@drawable/ic_plus"
+                app:tint="@color/wallet_text_color"
+                android:contentDescription="@null"
+                android:background="?android:attr/selectableItemBackground"
+                />
+
```

Comment:
nit: it probably would look better if the gap between plus and items can be reduced. Probably @kleantzogu might have some suggestions to improve the design, maybe different icon than the default figma one etc.

Reply:
We did some updates on Wallet 2.0. 
[Figma Link (Android)](https://www.figma.com/file/muVcmcu4RFNu0LkXe9VRDl/Wallet-2.0?type=design&node-id=5516-387547&mode=design&t=9ifWGAXggRfwrpia-4)
[Figma Prototype](https://www.figma.com/proto/muVcmcu4RFNu0LkXe9VRDl/Wallet-2.0?type=design&node-id=5164-359379&viewport=1178%2C-2210%2C0.28&t=9ifWGAXggRfwrpia-0&scaling=scale-down&starting-point-node-id=5516%3A389297&show-proto-sidebar=1)

Also attached is recording.

https://github.com/brave/brave-core/assets/2451237/b31876bc-e446-4a90-8962-3220d7e9e50a

Reply:
Thanks! Merging this as is for now and tweaking the UI in a follow-up PR.